src/modules/deposit/service.ts

```diff
@@ -222,6 +272,33 @@ export class DepositService {
       acx_rewards_amount_referee: d.acxRewardsAmountReferee,
     }));
   }
+
+  private _getFilteredDepositsQuery(
```

Comment:
I would not use the _ prefix since typescript already have a dedicated keyword for private

Reply:
https://github.com/across-protocol/scraper-api/pull/279/commits/a44ac9d35236845a1fc0a7d5ac196bb24c250ac3